packages/i18n/I18nDecorator/I18nDecorator.js

```diff
@@ -259,7 +313,7 @@ const contextDefaultConfig = {
 };
 
 const I18nContextDecorator = hoc(contextDefaultConfig, (config, Wrapped) => {
-	const {localeProp, rtlProp, updateLocaleProp} = config;
+	const {loadedProp, localeProp, rtlProp, updateLocaleProp} = config;
```

Comment:
Should we add this to `contextDefaultConfig`?  This doesn't appear to be used anywhere (is it?)... is it something we should document for future use or is it not needed for this ticket?